src/runner/main.cpp

```diff
@@ -23,13 +23,78 @@
 
 #include <winrt/Windows.System.h>
 
+#include <Psapi.h>
+#include <RestartManager.h>
+
 #if _DEBUG && _WIN64
 #include "unhandled_exception_handler.h"
 #endif
 #include <common/notifications/fancyzones_notifications.h>
 
 extern "C" IMAGE_DOS_HEADER __ImageBase;
 
+const wchar_t ExplorerProcessName[] = L"explorer.exe";
+const wchar_t PTUpdateMessageBoxTitle[] = L"PowerToys";
+const wchar_t PTUpdateMessageBoxText[] = L"PowerToys was updated and some components require Windows Explorer to restart. Do you want to restart Windows Explorer now?";
+
+void GetProcessInfo(const std::wstring& processName, std::vector<RM_UNIQUE_PROCESS>& processInfos)

```

Comment:
Let's move this function to `common` to some new `RestartAPI.cpp`. It could be internal and only `RestartProcess` could be exposed for now. Also let's return std::vector instead of using an outer parameter and rename it to something like `GetProcessInfoByName`

Reply:
Moved to common, `RestartManagement.h/cpp`